packages/create-remix/templates/deno/package.json

```diff
@@ -7,9 +7,6 @@
     "dev": "cross-env NODE_ENV=development remix build && run-p dev:*",
     "start": "cross-env NODE_ENV=production deno run --allow-net --allow-read --allow-env ./build/index.js"
   },
-  "dependencies": {
-    "@remix-run/deno": "*"
```

Comment:
`npm install` on a project created from the `deno` template fails for me with `Cannot find module '@remix-run/deno/package.json'`. I added `@remix-run/deno` as a dependency of my project as a workaround.

Reply:
Alternatively, if we want to forgo `@remix-run/deno`, we'd need to remove or alter the `postinstall` setup step in the project's `package.json`.